deps/build.jl

```diff
@@ -2,13 +2,16 @@ using BinDeps
 
 @BinDeps.setup
 
-blasvendor = Base.BLAS.vendor()
+using Libdl
```

Comment:
You may need Compat for Libdl, LinearAlgebra, and Sys

Reply:
Yep, but to be honest I don't know what to do with `NamedTuples`, as on `0.6` we get
```julia
julia> (x=1,)
ERROR: syntax: assignment not allowed inside tuple
```